cookbooks/aws-parallelcluster-slurm/attributes/slurm_attributes.rb

```diff
@@ -0,0 +1,4 @@
+# Slurm attributes shared between install_slurm and configure_slurm_accounting
+default['cluster']['slurm']['commit'] = ''
```

Comment:
question, non-blocking, what's the difference between commit and sha? are both needed?

Reply:
The sha is the checksum for the tarball we download from GitHub, not the commit hash.

Reply:
To add a few cent, the commit can be used to pick a very specific slurm version instead of an officially released one. Like for example to test a patch or a feature that is still in development.